### ISSUE TITLE
review and complete visible focus indicators

### ISSUE THREAD

**Issue by abdarslan** [bounswe/bounswe2025group5#637]
### Feature Description
Review the current status of the frontend and check if all the elements that user can interact has a focus indicator when it interacts either with a cursor or keyboard navigation. If there are missing ones add visual indicators for them as well.

### Why Is This Needed?
This is necessary for having user friendly, consistent ui and it is compliant with the accessibility requirements.

**Comment by abdarslan** [comment]
I've tested all ui both with keyboard and mouse for every single interactive element and they all have visual indicator for focus. So, we can close this issue and tick the relevant checkbox in the parent issue.
